src/patternfly/components/Progress/progress.scss

```diff
@@ -34,12 +35,17 @@
 
   // Modifiers
   // size modifiers
+  &.pf-m-sm {
+    --pf-c-progress__bar--Height: var(--pf-c-progress--m-sm__bar--Height); // set height to the small height
+  }
+
   &.pf-m-lg {
-    --pf-c-progress__bar--Height: var(--pf-c-progress__bar--Height--lg); // set height to the large height
+    --pf-c-progress__bar--Height: var(--pf-c-progress--m-lg__bar--Height); // set height to the large height
   }
 
-  &.pf-m-sm {
-    --pf-c-progress__bar--Height: var(--pf-c-progress__bar--Height--sm); // set height to the small height
+  // No Description or Status Modifier
+  &.pf-m-nodescription {
+    grid-gap: 0;
```

Comment:
Is it valid to have an outside measure and/or status an no description? If so then eliminating the grid gap means that they lose their spacing. Instead, I recommend adjusting the grid cells if there is nothing in the first row rather than eliminating the grid gap.

Reply:
@srambach This example was needed for a progress bar in one row without any description and spacing. If I only want one row is there another way to get that without removing the grid-gap? If I did what you suggest above there would still be 2 rows right?

Reply:
What I would suggest is a variation I guess, where if there would be nothing in the top row, then the grid-row definitions are changed so that there is only one row. Because removing the grid-gap would remove it everywhere, so if there is more than one cell ( like the progress bar and an outside measure) then they will be missing the grid gap.

Reply:
Sounds good @srambach. I made those changes if you want to review again :)